### ISSUE TITLE
install snakemake for running the .sh script

### ISSUE THREAD

**Issue by shreyaramachandran** [aabiddanda/bam_qc_pipeline#3]
I'm thinking that since installing snakemake needs a little more than just module load snakemake, it might be helpful to include that as part of the the run_snakemake.sh file?

**Comment by aabiddanda** [comment]
@shreyaramachandran good point! It might be useful to have it as a specific section in the README maybe (Issue #2)? One could envision a specific section that has the following lines to make it more explicit?

```
module load gcc/6.2.0 python/3.6.0
pip3 install snakemake --user
export PATH=$PATH:~/.local/bin/
```

Only reason I am suggesting this instead of in the `run_snakemake.sh` is because the above commands would only have to be run once when setting up your system, rather than every time you want to run your pipeline.

**Comment by aabiddanda** [comment]
@shreyaramachandran would you be able to add these lines into a section of the README file?
